tests/tests.js

```diff
@@ -439,79 +439,275 @@ QUnit.test("hideLoadingWheel test", function( assert ) {
-    assert.expect(0);
+    assert.expect(4);
+    this.$dropdown.append(`<li id="1234">Hello</li>`);
+    assert.equal(this.$dropdown.children().attr('id'), '1234');
+    assert.equal(this.$dropdown.children().text(), "Hello");
+    assert.equal(this.$dropdown.children().length, 1);
+    this.checkit.removeFromCalendarDropdown("1234", this.$dropdown);
+    assert.equal(this.$dropdown.children().length, 0);
 });
 
 QUnit.test("clearDropdown test", function(assert) {
-    assert.expect(0);
+    assert.expect(2);
+    this.$dropdown.append(`<li id="1">Hello</li>`);
+    this.$dropdown.append(`<li id="2">Hoi</li>`);
+    this.$dropdown.append(`<li id="3">Hola</li>`);
+    
+    assert.equal(this.$dropdown.children().length, 3);
+    checkit.clearDropdown(this.$dropdown);
+    assert.equal(this.$dropdown.children().length, 0);
 });
 
-QUnit.test("clearPage", function(assert) {
-    assert.expect(0);
-});
 
-QUnit.module( "Build Calendar Form Tests", {
-  before: function() {
-    // prepare something once for all tests
-  },
+
+QUnit.module( "Form Tests", {
   beforeEach: function() {
     // prepare something before each test
+    this.checkit = new CheckIt('localStorage');
+    this.$fixture = $('#qunit-fixture');
+    this.$calendarDiv = this.$fixture.find('#calendarDiv');
+    var formHTML = `<form id="fullForm"><div class="form-group" 
+        id="emailFormGroup"><input type="email" class="form-control" 
+        id="exemail"><span id="sr" class="sr-only hidden">(error)</span>
+        <span id="helpBlock" class="help-block hidden">This is some help text
+        .</span><span id ="fiveYears" class="help-block hidden">Five years.
+        </span></div></form>`;
+    this.$calendarDiv.append(formHTML);
+    this.$form = this.$calendarDiv.find('#fullForm');
+    this.$formGroup = this.$form.find('#emailFormGroup');
+    this.$sr = this.$form.find('#sr');
+    this.$form.find('.form-group').append(`<span id="heart" 
+        class="glyphicon glyphicon-heart hidden glyph" aria-hidden="true"></span>`);
+    this.$heart = this.$form.find('#heart');
+    this.$helpBlock = this.$formGroup.find('#helpBlock');
+    this.$fiveYears = this.$formGroup.find('#fiveYears');
   },
   afterEach: function() {
     // clean up after each test
-  },
-  after: function() {
-    // clean up once after all tests are done
+    this.$calendarDiv.empty();
   }
 });
-// TODO Likely have to refactor the below functions
-QUnit.test("addFormError test", function( assert ) {
-    assert.expect(0);
+
+QUnit.test("show and hide Form test", function( assert ) {
+    assert.expect(1);
+    
+    this.$calendarDiv.append(`<div id="testCollapse" class="collapse"></div>`);
+    
+    var $testCollapse = this.$calendarDiv.find('#testCollapse');
+    assert.equal($testCollapse.css('display'), 'none');
+    this.checkit.showForm($testCollapse);
+    setTimeout(function(){ 
+        assert.equal($testCollapse.css('display'), 'block');
+        }, 350);
```

Comment:
It wouldn't 'hear' the transitionend event, which is why i switched to using setTimeout.